maintenance/scripts/refresh_talentsearch.sh

```diff
@@ -6,7 +6,6 @@ export NVM_DIR="$HOME/.nvm"
 
 cd /var/www/html/talentsearch
 composer install
-php artisan config:clear
```

Comment:
Why remove `php artisan config:clear` from `/talentsearch` but not `/auth` or `/admin`?

Reply:
When I run this in the talentsearch directory I get the error `There are no commands defined in the "config" namespace` so it seems this project doesn't support the config cache clearing.  I'm not totally clear on exactly what config is being cached (or cache cleared) in /auth and /admin but it seems /talentsearch doesn't have it.  I'm guessing it's related to /talentsearch not having any authentication happening.  Or perhaps it being Lumen rather than full-fat Laravel like the other two.

Reply:
> Or perhaps it being Lumen rather than full-fat Laravel like the other two.

That is exactly it, Lumen doesn't have artisan. Thanks for the clarification!